### DISCUSSION TITLE
Is it possible to change the value of a filter when another filter is applied?

### DISCUSSION THREAD

**Discussion by hassyyy** [avo-hq/avo#859] [Help] [answered]
Let's say there are two filters - A & B
I want to change the value of B automatically when the value of A is changed

I have a similar use-case for fields as well
When a field(A) is changed, I want the value of a dependent field(B) to be changed automatically

**Comment by hassyyy** [comment]
@adrianthedev Is there any workaround to achieve this?

**Reply by adrianthedev** [reply]
Hey there. I took some time off with the family for Easter. 
I don't think that's possible at the moment. 

I could add something to allow that next week. 

Until then, can you gove a concrete example of the filters you want to change?
I want to get a better idea of the use case.

**Reply by hassyyy** [reply]
@adrianthedev Here is an example. Let's say I have two select fields: State & City
In create/edit page, I want to change the options & default value for City field based on the selected value in State field
Similarly, in filters, when the value of State is changed, I want to do one of the following:
- Clear the city field
- Set a value of city which belongs to the state
- Set the first city in the state 
- etc

Hope you enjoy the holidays 🥂

**Reply by adrianthedev** [reply]
Got it. Sounds good. I might be able to add something like that pretty quick. 


Thank you!

**Reply by hassyyy** [reply]
Awesome! That would be great 🙏

**Reply by adrianthedev** [reply]
Hey @hassyyy . Version 2.5 added this feature. It's called "reacting to filters".
[Docs](https://docs.avohq.io/2.0/filters.html#react-to-filters)
[Demo video](https://youtu.be/M2RsNPPFOio?t=219)

**Reply by hassyyy** [reply]
Awesome, that works 🙌

**Comment by hassyyy** [comment]
Is it possible to implement the same functionality with fields when creating/editing a record?

**Reply by adrianthedev** [reply]
I have my mind on that, but that's a totally different beast. A lot more edge cases to support. It deserves a different kind of attention.

I'm a bit weary to add something as I added here, which is a bit hack-ish, because devs will want to use it right away and will be difficult to manage. I want to add something good for that.

**Reply by hassyyy** [reply]
Makes sense. I will wait for that until it's perfectly cooked 💪

**Comment by hassyyy** [comment, marked as answer]
Solution ✅ 
> Hey @hassyyy . Version 2.5 added this feature. It's called "reacting to filters". [Docs](https://docs.avohq.io/2.0/filters.html#react-to-filters) [Demo video](https://youtu.be/M2RsNPPFOio?t=219)
